data/base/script/campaign/cam3-2.js

```diff
@@ -14,8 +14,8 @@ const NEXUS_RES = [
 	"R-Sys-NEXUSrepair", "R-Wpn-Flamer-Damage06",
 ];
 var edgeMapIndex;
```

Comment:
`edgeMapIndex` is never used or initialized

Reply:
Sure.

Reply:
deleted